packages/react-components/src/templates/AnalyticsEngagementPageBody.tsx

```diff
@@ -70,8 +89,28 @@ const AnalyticsEngagementPageBody: React.FC<
       href={analytics({}).engagement({}).$}
       exportResults={exportResults}
     />
+    <CaptionCard>
+      <>
+        <CaptionItem label="Presentations" {...performance.events} />
```

Comment:
```suggestion
        <CaptionItem label="Events" {...performance.events} />
```

I think we could call 'Events' here as well

<img width="1720" alt="Screenshot 2024-09-06 at 07 54 37" src="https://github.com/user-attachments/assets/a4f3021b-997c-4d09-ae7c-e2d730dfa38d">